### ISSUE TITLE
How many Epochs will be Run in Segmentation Example in Develop branch

### ISSUE THREAD

**Issue by picheny-nyu** [pyannote/pyannote-audio#693]
I am trying to use the example for Segmentation in the develop branch. How many epochs will be run? If I understand correctly the number will be 1000 as there don't seem to be any early stopping criteria specified. Is that correct?

**Comment by hbredin** [comment]
That is correct. 

See [this](https://github.com/pyannote/pyannote-audio/blob/develop/pyannote/audio/cli/train.py) training script (not yet documented) for inspiration on how to [add early stopping](https://github.com/pyannote/pyannote-audio/blob/2ecdc23c2129a132d5ef7432182a9619730f02ad/pyannote/audio/cli/train.py#L140-L148).

**Comment by picheny-nyu** [comment]
It looks like validation loss needs to be added to the PyanNet.py model. Is this correct?

**Comment by picheny-nyu** [comment]
Hi there....can anyone help me with the above, I am having some trouble figuring this out.
